joomla-extensions/weblinks
# Improved the Batch view of com_weblinks

Pull Request for Issue https://github.com/joomla-extensions/weblinks/issues/367.

### Summary of Changes
I changed the batch view of com_weblinks analogously to the batch view of com_content.


### Testing Instructions
1. Create a few weblinks via com_weblinks (Components | Web Links).
2. Select all this weblinks and click the toolbar button batch
3. See that the view has a nice margin.
4. Please also check that the component is still working correctly. For example: Move weblinks to another category or change the tags.
